stdlib/LinearAlgebra/src/bidiag.jl

```diff
@@ -184,6 +184,10 @@ end
     return x
 end
 
+Base._reverse(A::Bidiagonal, dims) = reverse!(Matrix(A); dims)
+Base._reverse(A::Bidiagonal, dims::Colon) = Bidiagonal(reverse(A.dv), reverse(A.ev), A.uplo)
+Base._reverse!(A::Bidiagonal, dims::Colon) = (reverse!(A.dv); reverse!(A.ev); A)
```

Comment:
```suggestion
```
This method should be deleted — `reverse!` on a `Bidiagonal` matrix is an error, because you can't change the `uplo` in-place.